app/composables/useAuth.ts

```
@@ -69,14 +69,27 @@ export function useAuth() {
-      retry: 0
-    })
+    let data: { session?: any, user?: any } | null = null
+    try {
+      if (import.meta.client) {
+        data = await $fetch('/api/auth/get-session', {
+          credentials: 'include',
+          headers
+        })
+      } else {
+        // Use useFetch for better SSR support and hydration
+        const { data: sessionData } = await useFetch<{ session?: any, user?: any }>('/api/auth/get-session', {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)